tests/integration/test_case.go

```diff
@@ -205,6 +205,9 @@ type CreateDoc struct {
 	// If a value is not provided the document will be created in all nodes.
 	NodeID immutable.Option[int]
 
+	// The identity of this request.
+	Identity string
```

Comment:
suggestion: I think `immutable.Option[string]` would be a better type for these.

todo: Please document for what this means/does when it is and isnt provided (e.g. when provided the created doc will be owned by this identity and will need to respect any collection-level policies, if none, the document will be public, etc)

For all the actions now hosting this property (Update/Delete/Req)

Reply:
It might be a better to make it optional, but these will likely be re-worked when authentication stuff makes the identity type not a string anymore, will think and make optional then. As for documentation I can do for sure.

- [ ] Add more documentation to the identity test args.